patches/chrome-browser-google-google_update_win.cc.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/chrome/browser/google/google_update_win.cc b/chrome/browser/google/google_update_win.cc
+index 6a1eda7280eeb492e6f3ed874b85840d46a61200..b805a2bb05a5424f72e9fb51e9f0f8fe952c221f 100644
+--- a/chrome/browser/google/google_update_win.cc
++++ b/chrome/browser/google/google_update_win.cc
+@@ -838,10 +838,15 @@ void UpdateCheckDriver::OnUpgradeError(GoogleUpdateErrorCode error_code,
+   }
+ 
+   base::string16 html_error_msg =
++#if defined(BRAVE_CHROMIUM_BUILD)
++      // We don't have links for each errors.Only shows error code.
++      base::StringPrintf(L"%d", error_code_);
++#else
+       base::StringPrintf(L"%d: <a href='%ls0x%X' target=_blank>0x%X</a>",
+                          error_code_, base::UTF8ToUTF16(
+                              chrome::kUpgradeHelpCenterBaseURL).c_str(),
```

Comment:
we don't we just change kUpgradeHelpCenterBaseURL to `"https://community.brave.com/?error=";

Reply:
I thought the original change had something like that, but now I can't see it anymore

Reply:
Right. `chrome::kUpgradeHelpCenterBaseURL`  in `url_constants.cc` was modified like that.
However,  that change breaks this file's rebase strategy(https://github.com/brave/brave-core/pull/155)
Also, we don't have a plan for providing specific links for specific error code for now.
So, link for specific error code was removed.

Reply:
I don't understand what you mean about the rebase strategy. I would rather pass an error code that we don't use yet than patch to remove it

Reply:
Updated. Please check again!